test/plugins/pipelines.test.js

```diff
@@ -919,15 +919,11 @@ describe('pipeline plugin test', () => {
         });
 
         it('formats the checkout url correctly', () => {
-            const goodCheckoutUrl = 'git@github.com:screwdriver-cd/data-model.git#master';
+            userMock.getPermissions.withArgs(checkoutUrl).resolves({ admin: false });
```

Comment:
getPermissions will be called with `scmUri` instead of the `checkoutUrl`
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/pipelines/create.js#L61